wrappers/fedimint-kotlin/src/main/kotlin/com/cashu/Application.kt

```diff
@@ -0,0 +1,154 @@
+package com.cashu
+
+import io.github.cdimascio.dotenv.Dotenv
+import kotlinx.coroutines.runBlocking
+import java.lang.Exception
+import kotlin.system.exitProcess
+import io.github.cdimascio.dotenv.dotenv
+
+fun main() {
+    try {
+        val dotenv = dotenv {
+            // Provide an absolute path to the .env file
+            directory = "/Absolute/path/to/file/.env"
+            // ignoreIfMalformed = true
+             ignoreIfMissing = true
+        }
```

Comment:
This seems weird for dotenv, it shouldn't have to be passed a dir it should just pick it up from where it's run